tests/network/conftest.py

```diff
@@ -312,12 +319,38 @@ def _verify_ipv4():
             else:
                 LOGGER.info("Validated network lane is running against an IPV4 supported cluster")
 
+    def _verify_bgp_env_vars():
+        """Verify if the cluster supports running BGP tests.
+
+        Requires the following environment variables to be set:
+        BR_EX_VLAN_TAG: expected VLAN number on the node br-ex interface.
+        EXTERNAL_FRR_STATIC_IPV4: reserved IP in CIDR format for the external FRR pod inside BR_EX_VLAN_TAG network.
+        BGP_CLUSTER_DOMAIN_GROUP: cluster domain group.
+        """
+        if any(test.get_closest_marker("bgp") for test in collected_tests):
+            LOGGER.info("Verifying if the cluster supports running BGP tests...")
+            required_env_vars = ["BR_EX_VLAN_TAG", "EXTERNAL_FRR_STATIC_IPV4", "BGP_CLUSTER_DOMAIN_GROUP"]
```

Comment:
Continuing the previous "lost" thread [1]:

Can you please feedback?

[1] https://github.com/RedHatQE/openshift-virtualization-tests/pull/1776#discussion_r2330404868